.github/DEV_GUIDE.md

```diff
@@ -52,6 +52,7 @@ If this does not work, search for the following extensions:
  - GitHub Pull Requests - lets you make a pull request directly from the editor
  - Bracket Pair Colorizer 2 - paints matching brackets the same color
  - Git Graph - lets you work with branches visually for ease of use
+ - ESLint - lets you see some code errors in your editor as you work, also auto-fixes code style items on save
```

Comment:
ok

Reply:
I'm actually a little torn on fix-lint-errors-on-save, but we'll see how it goes.